packages/core/src/types.ts

```diff
@@ -886,10 +921,20 @@ export enum ActionTypes {
   Choose = 'xstate.choose'
 }
 
-export interface RaiseAction<TEvent extends EventObject> {
+// Used as a "phantom type" for built-in actions
+export interface BuiltInActionObject<TContext, TEvent extends EventObject> {
+  type: string;
+  __context: TContext;
+  __event: TEvent;
+}
+
+export type RaiseAction<TEvent extends EventObject> = BuiltInActionObject<
+  any,
```

Comment:
After some recent type-related problems in the viz I was wondering if it wouldnt be better to use `unknown` instead of `any` in situations like this. WDYT? With `any` it's easy to spill it somewhere unknowingly and get runtime errors because one thought that some property access was safe - it type "checked" after all ;p

Reply:
Done 👍